src/tiledimage.js

```diff
@@ -660,6 +667,8 @@ $.extend($.TiledImage.prototype, $.EventSource.prototype, /** @lends OpenSeadrag
         $.console.assert(!newClip || newClip instanceof $.Rect,
             "[TiledImage.setClip] newClip must be an OpenSeadragon.Rect or null");
 
+//TODO: should this._raiseBoundsChange(); be called?
```

Comment:
Agreed... if getBounds isn't changing, `bounds-change` shouldn't be raised. If the World updates its home bounds based on the clip change, we should have a separate `clip-change` event that it can watch.